ethereum/sidecar/cli/flags.go

```diff
@@ -12,6 +13,8 @@ const (
 	FlagServerNetwork             = "ethereum-sidecar.server.network"
 	FlagServerBatchSize           = "ethereum-sidecar.server.batch-size"
 	FlagServerRequestsPerMinute   = "ethereum-sidecar.server.requests-per-minute"
+	FlagBridgeOutServerAddress    = "ethereum-sidecar.bridge-out.server-address"
+	FlagBridgeOutRequestTimeout   = "ethereum-sidecar.bridge-out.request-timeout"
```

Comment:
I'm not sure if we should make this timeout configurable. I don't see any reason an operator would want to change that to be honest.